templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -302,6 +302,29 @@
 			</div>
 		</div>
 
+		{{if or .HasIssuesOrPullsWritePermission .Issue.IsPrivate .IsPoster}}
+		<div class="ui divider"></div>
+		<div class="ui">
+			<span class="text"><strong>{{.i18n.Tr "repo.issues.confidential"}}</strong></span>
+			<div class="mt-3">
+				<form method="POST" action="{{.Issue.Link}}/private">
+					<input type="hidden" name="is_confidential" value="{{if .Issue.IsPrivate}}false{{else}}true{{end}}" />
+					{{$.CsrfTokenHtml}}
+					<button class="fluid ui button df jc">
+						{{if .Issue.IsPrivate}}
+							<!-- TODO: Replace it with a better icon! --> 
+							{{svg "octicon-mute" 16 "mr-3"}}
+							{{.i18n.Tr "repo.issues.confidential.off"}}
+						{{else}}
```

Comment:
I wouldn't even display an icon if the issue is public.
I'd say that confuses more than it helps.
You can quickly see if an issue is confidential or not simply by looking if the `confidential` icon is displayed.